src/backends/kafka/mod.rs

```diff
@@ -364,13 +381,43 @@ mod tests {
         consumer.subscribe(&[topic], my_callbacks).unwrap();
     }
 
+    #[tokio::test]
+    async fn test_tell() {
+        create_topic("test", 1).await;
+        let configuration = KafkaConfig::new_consumer_config(
+            vec!["localhost:9092".to_string()],
+            "my-group-1".to_string(),
+            "latest".to_string(),
+            false,
+            None,
+        );
```

Comment:
I think assignment callback is properly implemented in the local consumer.
Was there a specific reason to rely on a real Kafka consumer for this? It makes tests slower and depending on Kafka to be actually working, I think we should avoid that when possible.
Or do you actually want to test the specific Kafka implementation behavior.

Reply:
Yeah, I do want to test against real Kafka even though it is slow. Are you suggesting actually skipping all the tests for this module?

Reply:
Sorry. I did not realize this was in the kafka module. 
No, testing the Kafka module against Kafka is good.